antarest/study/storage/variantstudy/model/command/create_link.py

```diff
@@ -211,13 +140,16 @@ def _apply(self, study_data: FileStudy, listener: Optional[ICommandListener] = N
         output, data = self._apply_config(study_data.config)
         if not output.status:
             return output
+
+        validated_properties = LinkInternal.model_validate(self.parameters).model_dump(
```

Comment:
I think you should add filters to the excludes here depending on the version as its seems you always write them into the file